packages/flutter/lib/src/material/pickers/input_date_picker.dart

```diff
@@ -220,29 +221,26 @@ class _InputDatePickerFormFieldState extends State<InputDatePickerFormField> {
 
   @override
   Widget build(BuildContext context) {
-    return OrientationBuilder(builder: (BuildContext context, Orientation orientation) {
-      assert(orientation != null);
-
-      return TextFormField(
-        decoration: InputDecoration(
-          border: const UnderlineInputBorder(),
-          filled: true,
-          // TODO(darrenaustin): localize 'mm/dd/yyyy' and 'Enter Date'
-          hintText: widget.fieldHintText ?? 'mm/dd/yyyy',
-          labelText: widget.fieldLabelText ?? 'Enter Date',
-        ),
-        validator: _validateDate,
-        inputFormatters: <TextInputFormatter>[
-          // TODO(darrenaustin): localize date separator '/'
-          DateTextInputFormatter('/'),
-        ],
-        keyboardType: TextInputType.datetime,
-        onSaved: _handleSaved,
-        onFieldSubmitted: _handleSubmitted,
-        autofocus: widget.autofocus,
-        controller: _controller,
-      );
-    });
+    final InputDecorationTheme inputTheme = Theme.of(context).inputDecorationTheme;
```

Comment:
Is there a plan to add inputDecoration as param to the picker theme once that is created? For internal use, we would need that functionality, so that the picker can have a different input decoration than the theme provided one.

Reply:
Yeah, I hadn't thought much about it yet, but that would make sense. If one exists in the picker theme it would be used, otherwise we would use the logic in this PR.